src/handlers/predict_torch_sequence.py

```diff
@@ -0,0 +1,77 @@
+import base64
+import onnxruntime
+import torch
+import io
+import numpy as np
+import torch.nn.functional as f
+from jaqpotpy.descriptors.tokenizer import SmilesVectorizer
+from jaqpotpy.api.openapi import ModelType, PredictionRequest, PredictionResponse
+from jaqpotpy.descriptors.graph.graph_featurizer import SmilesGraphFeaturizer
+from src.helpers.utils import to_numpy
+
+
+def torch_sequence_post_handler(request: PredictionRequest) -> PredictionResponse:
+    feat_config = request.model.torch_config
+    featurizer = _load_featurizer(feat_config)
+    target_name = request.model.dependent_features[0].name
+    model_task = request.model.task
+    user_input = request.dataset.input
+    raw_model = request.model.raw_model
+    predictions = []
+    for inp in user_input:
+        print(featurizer.transform([inp["SMILES"]]))
+        model_output = onnx_post_handler(
+            raw_model, featurizer.transform(featurizer.transform([inp["SMILES"]]))
+        )
+        predictions.append(check_model_task(model_task, target_name, model_output, inp))
```

Comment:
The `check_model_task` method implies that it's a boolean function that returns true or false. In here the method returns the predictions based on the task, so maybe rename the method to `generate_prediction_response` so it's clearer to the reader what the method does?